docs/why_mf_next.md

```diff
@@ -108,23 +130,20 @@ localization flow:
 
 ### 4. Designed to be API only, plain text, UI, “imperative style”
 
-The main (only?) use case for `MessageFormat` is: load the string from resources,
-replace placeholders, and return the string result with placeholders replaced. \
+The typical use case for `MessageFormat` is: load the string from resources,
+replace placeholders, and return the string result with placeholders replaced.
 An i18n-aware `printf`, basically.
 
-It does not play well with binding, formatting tags (think `html`),
+It does not account for formatting tags (such as HTML),
 or “document-like” content (for example templating systems like
 [freemarker](https://freemarker.apache.org/),
 [mustache](https://mustache.github.io/), even JSP, PHP, etc.)
 
-Because it is API only it has no standard way to store the stings in a
-serialized format and to carry info or directives for translators or
-localization tools. \
-So there is no way for a message to reference another message, or to fallback
-to a different locale. That is all left to the "host resource manager"
-(whatever that is for the given tech stack)
+Because it is only an API, it has no standard way to store strings in a
+serialized format or to carry information or directives for translators
+or localization tools.
 
-There is also no metadata: comments, length limits, example, links,
+There is also no metadata “packaging”: comments, length limits, example, links,
```

Comment:
This is a duplicate of the above; should be combined. (There are other duplicates in the text, mostly harmless and would take additional work to fix; given that we want to get this out pronto I didn't suggest changes.)